odata-json-format/10 Media Entity.md

```diff
@@ -175,7 +175,7 @@ control information MUST be included in a response that represents a
 partial result.
 
 ::: example
-Example 28:
+Example ##ex:
```

Comment:
We must bring this correction into CSD01.

Reply:
Then let's move on this in the next TC call